bogota-july-2018/Calculator-JS/DaniloPlazas/Calculator HTML/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
Minor: Not sure about the tab spacing you are using, by default is ok to use 4 spaces to indent the files